src/components/Plot/Plot.sass

```diff
@@ -36,8 +45,11 @@ $not-allowed-color: rgba(255, 0, 0, 0.5)
       cursor: pointer
       border-color: var(--color-yellow)
 
+    &.is-in-hover-range
+      border-color: var(--color-yellow)
+
   .cleanup-mode &.crop,
-    background-color: rgba(255, 255, 0, 0.5)
+    background-color: var(--color-yellow)
```

Comment:
this comment applies to this whole file and how these CSS vars are used because I was using them this same way up until just before the recent holiday break, but then came to realize that this is still a "sassy" approach to them. CSS vars are most powerful for dynamic values, for static uses a SASS var is ultimately more performant.

so, instead of having these CSS vars like `--color-yellow` and `--color-blue-water`, they could instead be one var like `--plot-background-color`, which gets redefined in certain situations. the value could however come from SASS vars, to really make the most of everything available to us.

```sass
// these might be in a different file
$color_yellow: rgba(255, 255, 0, 0.5)
$color_blue_water: rgba(125, 245, 255, 0.75)

.Plot
    background-color: var(--plot-background-color)

.water-mode &.crop:hover
    --plot-background-color: #{$color_blue_water}

.cleanup-mode &.crop,
    --plot-background-color: #{$color_yellow}
```

this is just an example with one property, though there's a few that I see could be updated. it's also a little bit of a tangent from the intention of this PR so I don't think it _has_ to change to merge, but I wanted to mention it while I was thinking of it. I'm also happy to help update this file if you want!

Reply:
I see what you're saying! You have a great point; Sass variables make a lot more sense for static values like this. I made the change in https://github.com/jeremyckahn/farmhand/commit/9e7e84a2b761c9249e613721164d352973bad5ea. Is that what you had in mind?